OxfordBodleian/Juel-Jensen/BDLaethe33.xml

```diff
@@ -218,39 +258,88 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-                           
+                                <seg type="script"/>
+                                <locus target="#1r"/>
+                                <seg type="ink">Black, red</seg>
+                                <date notBefore="1700" notAfter="1799">18th century</date>
+                                <seg type="rubrication">
+                                    The name of Giyorgis; the word <foreign xml:lang="gez">ሃሌ፡</foreign>; elements of the punctuation signs and numerals.
+                                </seg>
+                                <list type="abbreviations">    
+                                    <item>
+                                        <abbr>ሃሌ፡</abbr>, for <expan>ሃሌ፡ ሉያ፡</expan> (<locus target="#1r"/>)
+                                    </item>
+                                </list>
+                            </handNote>
+                            <handNote xml:id="h2" script="Ethiopic">
+                                <locus from="2r" to="130v"/>
+                                <seg type="script"/>
+                                <seg type="ink">Black, red</seg>
+                                <date notBefore="1700" notAfter="1799">18th century</date>
+                                <desc>
+                                    Small and occasionally irregular handwriting.
+                                </desc>
+                                <seg type="rubrication">
+                                    Divine names; number and titles of the psalms; name, number, and traditional interpretation of the Hebrew letter in Ps. 118; 
+                                    elements of the punctuation signs and numerals.
+                                </seg>
                             </handNote>
                         </handDesc>
                         
                        <!-- <decoDesc>
-                         
                         </decoDesc>-->
                         
-                     <!--   <bindingDesc>
-                            <binding xml:id="binding">
-                                <decoNote xml:id="b1"></decoNote>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <desc type="MagicText"/>  
+                                    <title ref="LIT4625Prayer"/>
+                                    <locus target="#18r"/>
+                                    <note> 
+                                        The note is crudely written in a pale black ink and is incomplete. Many words are not fully understandable. It starts as follows:
+                                        <q xml:lang="gez">
```

Comment:
New additions mark up possibilities! https://betamasaheft.eu/Guidelines/?id=additionsVaria
For incipits, you can use `<seg part="I">` inside `<q>`

Reply:
So, do `<seg part="I">` and `<seg part="F">` stand for incipit and explicit respectively? Why "F"?

Reply:
Something with initial and final, I think...